src/manifests/test_manifest.py

```diff
@@ -36,6 +36,9 @@ class TestManifest(ComponentManifest['TestManifest', 'TestComponents']):
               test-configs:
                 - with-security
                 - without-security
+            smoke-test:
+              test-spec:
```

Comment:
Do we want to keep this as `test-configs`? Similar to integ-test and bwc test?

Reply:
I'm thinking it's not technically config? Keep it as `test-configs` may cause some confusion to users since we don't modify any of the cluster configuration.